forge-game/src/main/java/forge/game/spellability/SpellAbility.java

```diff
@@ -2068,7 +2068,7 @@ public boolean isTargeting(GameObject o) {
         if (getTargets().contains(o)) {
             return true;
         }
-        SpellAbility p = getParent();
+        SpellAbility p = getSubAbility();
```

Comment:
You changed the Logic from going Up to the Parent
to go down to the Sub Abilities instead

i don't know if that would have other consequences

Especially if other functions still do the targeting in the other direction

Reply:
I know, I checked all places it gets called and this only seems to fix corner cases
e. g. _Torchling_ wasn't able to target entwined _Abuna's Chant_